__tests__/e2e/trees.spec.ts

```diff
@@ -135,4 +135,16 @@ describe('trees', () => {
     },
     1000 * 30,
   );
+
+  it(
+    'trees?lat=39.0619&lon=-122.6064&lat=38.1519&lon=-123.3139&lat=37.8879&lon=-121.1001',
```

Comment:
@dagmawig we need you the standard data format for this, which is the geojson, so you need to give a parameter like `geojson` following the protocol: https://geojson.org/

Reply:
And also, postgresql recogonize geojson for example this: https://postgis.net/docs/ST_GeomFromGeoJSON.html